docs/docs/clients/client-side/nextjs-and-ssr.md

```diff
@@ -48,18 +48,17 @@ state.
 
 ```javascript
 import { FlagsmithProvider } from 'flagsmith/react';
-import flagsmith, { createFlagsmithInstance } from 'flagsmith/isomorphic';
+import { createFlagsmithInstance } from 'flagsmith/isomorphic';
 function MyApp({ Component, pageProps, flagsmithState }) {
+ const flagsmithRef = useRef(createFlagsmithInstance());
```

Comment:
@kyle-ssg - Would I need to add the UUID of the user in here? or is it fine to use as is?